### PR TITLE
Fix #281: Get rid of the global current pressure state

### LINKED ISSUE #281: "Current pressure state" definition and cardinality are confusing
https://w3c.github.io/compute-pressure/#dfn-current-pressure-state is currently defined as a per-global string.

However:
- Its value is not set anywhere in the spec.
- All locations in https://w3c.github.io/compute-pressure/#contributing-factors that reference/read its value do not provide a corresponding global.
- The way it is referenced in https://w3c.github.io/compute-pressure/#dfn-state makes it sound like this concept is per-source type, not per global, at the very least.

### PR DESCRIPTION
**Pull request by kenchris**
***
<a href="https://pr-preview.s3.amazonaws.com/kenchris/compute-pressure/pull/292.html" title="Last updated on Sep 19, 2024, 10:24 AM UTC (2628ae1)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/compute-pressure/292/2e54dab...kenchris:2628ae1.html" title="Last updated on Sep 19, 2024, 10:24 AM UTC (2628ae1)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1227,8 +1224,7 @@ <h3>Supporting algorithms</h3>
           Let |record:PressureRecord| be |observer|.{{PressureObserver/[[LastRecordMap]]}}[|source|].
         </li>
         <li>
-          If |record|.{{PressureRecord/[[State]]}} is not equal to |state| and [=change in contributing factors is substantial=]
```
Comment (rakuco) [inline index.html:1230]: Is this change related to the rest? It doesn't seem to reference `[=current pressure state=]`.

Comment (kenchris) [reply index.html:1230]: It is. This is done on the global value now instead of per observer.

Comment (rakuco) [reply index.html:1230]: I see. It'd be good to explain this in the commit/PR message, as it's not immediately clear just by looking at the patch.

**Review by rakuco — APPROVED** [review]
lgtm modulo the explanation comment
